src/services/chat/prepopulated.js

```diff
@@ -41,15 +41,29 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser.defaulValues
+                  updated_at: ownUser?.updated_at,
+                  is_anonymous: ownUser?.is_anonymous,
+                  name: ownUser?.username,
+                  image: ownUser?.profile_pic_path,
+                  last_active: ownUser?.last_active_at,
+                  username: ownUser?.username,
+                  id: ownUser?.user_id,
+                  created_at: ownUser?.created_at
                 },
                 user_id: ownUser?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
               },
               {
                 user: {
-                  ...user.defaulValues
+                  updated_at: user?.updated_at,
+                  is_anonymous: user?.is_anonymous,
+                  name: user?.username,
+                  image: user?.profile_pic_path,
+                  last_active: user?.last_active_at,
+                  username: user?.username,
+                  id: user?.user_id,
+                  created_at: user?.created_at
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar issues as above for the `user` object. The `name` property is being assigned the value of `user?.username`, which might not be correct if `name` and `username` are different properties in the `user` object. Also, the `is_anonymous` property is hardcoded to `false`.
```diff
- name: user?.username,
+ name: user?.name,

- is_anonymous: false,
+ is_anonymous: user?.is_anonymous,
```

Please ensure these changes align with your intended logic and data structure.


<!-- This is an auto-generated comment by OSS CodeRabbit -->